### PR TITLE
Pvc capacity report

### PR DESCRIPTION
**Pull request by EmilyM1**

### CODE REVIEW

**Comment by EmilyM1** [comment]
/(pony)

File: test/integration/main_test.go [hunk 1]
```diff
@@ -190,6 +190,19 @@ var (
 			comparisonColumnNames:        []string{"persistentvolumeclaim_usage_bytes"},
 			timeout:                      reportTestTimeout,
 		},
+		{
+			name:      "persistentvolumeclaim-capacity",
+			queryName: "persistentvolumeclaim-capacity",
+			dataSources: []testDatasource{
+				{
+					DatasourceName: "persistentvolumeclaim-capacity",
```
Comment (chancez) [inline test/integration/main_test.go]: This needs to be `persistentvolumeclaim-capacity`

**Pull request merged**